repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -77,6 +78,31 @@ class Interpreter(shellPrompt0: Ref[String],
       }
     }
   }
+  
+  def processModule(code: String): Unit = {
```

Comment:
This fella has a whole lot of duplicate code with `processScript` above; let's pull it out